galeracluster/source/documentation/mysql-wsrep-options.rst

```diff
@@ -1260,7 +1260,7 @@ Defines the Online Schema Upgrade method the node uses to replicate :abbr:`DDL (
    "Dynamic Variable", "Yes"
    "Permitted Values", "Enumeration"
    "Default Value", "``TOI``"
-   "Valid Values", "``TOI``, ``RSU``", ``NBO``"
+   "Valid Values", "``TOI``, ``RSU``, ``NBO``"
    "Initial Version", "MySQL-wsrep: 5.5.17-22.3, MariaDB: 5.5.21"
    "Initial Version, NBO", "MariaDB Enterprise Server Version 10.5, Percona XtraDB Cluster 8.0.25-15.1"
```

Comment:
We have made no mention of NBO in Galera Cluster Enterprise Edition (EE)? We must

Reply:
How do we do this?